packages/api/src/routers/profiles.ts

```
@@ -0,0 +1,387 @@
+
+        return {
+          success: true,
+          data: newRating,
+          message: "Rating submitted successfully",
+        };
+      });
+      return {
+        success: true,
+        data: newRating,
+        message: "Rating submitted successfully",
+      };
```

Biome (2.1.2)

[error] This code will never be reached ...

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)